mesonbuild/interpreter/type_checking.py

```diff
@@ -144,8 +144,6 @@ def variables_validator(contents: T.Union[str, T.List[str], T.Dict[str, str]]) -
     for k, v in variables.items():
         if not k:
             return 'empty variable name'
-        if not v:
-            return 'empty variable value'
```

Comment:
If we're going to add support for this, it should print a FeatureNew...

(Is this somehow needed for cargo?)

Reply:
Yes, cargo will do `declare_dependency(..., variables: {'features': 'f1,f2,...'})` to tell which features has been enabled when configuring that cargo subproject. That list could be empty.

Added FeatureNew.